src/sentry/web/frontend/demo_start.py

```diff
@@ -34,45 +17,14 @@ def post(self, request):
         if not settings.DEMO_MODE:
             raise Http404
 
-        # TODO: add way to ensure we generate unique petnames
-        name = generate_random_name()
-
-        slug = slugify(name)
-
-        email = create_fake_email(slug, "demo")
-        user = User.objects.create(
-            email=email,
-            username=email,
-            is_managed=True,
-            flags=User.flags["demo_mode"],
-        )
-
-        org = Organization.objects.create(
-            name=name,
-            slug=slug,
-            flags=Organization.flags["demo_mode"],
-        )
-        team = org.team_set.create(name=org.name)
-
-        owner = User.objects.get(email=settings.DEMO_ORG_OWNER_EMAIL)
-        OrganizationMember.objects.create(organization=org, user=owner, role=roles.get_top_dog().id)
-
-        member = OrganizationMember.objects.create(organization=org, user=user, role="member")
-        OrganizationMemberTeam.objects.create(team=team, organizationmember=member, is_active=True)
-
-        python_project = Project.objects.create(name="Python", organization=org, platform="python")
-        python_project.add_team(team)
-
-        reat_project = Project.objects.create(
-            name="React", organization=org, platform="javascript-react"
-        )
-        reat_project.add_team(team)
-
-        populate_python_project(python_project)
-        populate_react_project(reat_project)
+        from sentry.demo.demo_org_manager import assign_demo_org
```

Comment:
Let's leave a comment explaining why we're doing this, so that someone doesn't decide to just move it back out.